core/platform/email/mailgun_email_services.py

```diff
@@ -46,9 +48,9 @@ def post_to_mailgun(data):
     header = {'Authorization': auth_str}
     server = (
         'https://api.mailgun.net/v3/%s/messages' % feconf.MAILGUN_DOMAIN_NAME)
-    data = urllib.urlencode(data)
-    req = urllib2.Request(server, data, header)
-    return urllib2.urlopen(req)
+    data = python_utils.url_encode(data, doseq=False)
```

Comment:
Why the newly-added arg?

How did you test this manually?

Reply:
The default value of doseq is False(in urllib.urlencode) so there won't be any change in behaviour. I added this arg as some existing urrlib.urlencode is using this arg.

I added test for this function in python_utils_test.

Reply:
If the arg has a default value, and you are using the default value, then you do not need to specify it here (in the caller).

Reply:
Done